src/app/services/authentication.service.ts

```diff
@@ -0,0 +1,38 @@
+import {Injectable} from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+import {BehaviorSubject, Observable} from 'rxjs';
+import {map} from 'rxjs/operators';
+
+import {User} from '../models/user';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthenticationService {
+  baseUrl = 'https://good-deed.herokuapp.com';
```

Comment:
should be moved to constant outside the class

Reply:
Done